### ISSUE TITLE
Support EDuke32 

### ISSUE THREAD

**Issue by clobber** [jmarshall23/ShadowWarrior#2]

**Comment by jmarshall-blackenmace** [comment]
I agree. I set up the Visual Studio solution because I hate manually editing make files. Its on my todo list to integrate over the make files for other platforms.

**Comment by neuromancer** [comment]
Any ETA? I will be happy to test the makefile in ArchLinux when you need it.. 😄
